engine/jobmaster/dm/task_manager.go

```diff
@@ -115,6 +115,7 @@ func (tm *TaskManager) UpdateTaskStatus(taskStatus runtime.TaskStatus) {
 		zap.Stringer("unit", taskStatus.Unit),
 		zap.Uint64("config_modify_revison", taskStatus.CfgModRevision),
 	)
+	taskStatus.StageUpdatedTime = time.Now()
```

Comment:
Maybe we can save it somewhere other than taskStatus, so we don't have to add unnecessary properties to TaskStatus and change the unit test

Reply:
I prefer Stage and the StageUpdateTime to be together because they have tight relationship, will think about it soon